dev_guide/builds/build_troubleshooting.adoc

```diff
@@ -0,0 +1,26 @@
+[[dev-guide-build-troubleshooting]]
+= Build Troubleshooting
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:prewrap!:
+
+[cols="1,4",options="header"]
+.Troubleshooting Guidance for Builds
+|===
+|Issue |Resolution
+a|A build fails with:
+
+----
+requested access to the resource is denied
+----
+a|You have exceeded one of the xref:../../dev_guide/compute_resources.adoc#dev-guide-compute-resources[image
+quotas] set on your project. Check your current quota and verify the limits
+applied and storage in use:
+
+----
+$ oc describe quota
+----
+|===
```

Comment:
I know this is like this currently in the docs, but would it be better to take this out of a table and into an actual section? I think a table with just one thing is a little redundant, plus, having it like this isn't really consistent with the rest of the docs.